From 21dae85cfe1a7a50b24f0cb3071c6b7dbcf83e52 Mon Sep 17 00:00:00 2001
From: Stefan Hajnoczi <stefanha@redhat.com>
Date: Fri, 11 Jul 2014 17:18:29 +0200
Subject: [PATCH 41/44] simpletrace: install simpletrace.py

RH-Author: Stefan Hajnoczi <stefanha@redhat.com>
Message-id: <1405099110-10975-7-git-send-email-stefanha@redhat.com>
Patchwork-id: 59864
O-Subject: [RHEL6.6 qemu-kvm PATCH 6/7] simpletrace: install simpletrace.py
Bugzilla: 905761
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

The simpletrace.py script can be used as a Python module to implement
custom trace analysis scripts.  It can also be used stand-alone to
pretty-print trace files.

Install and ship simpletrace.py so it can be used to analyze SystemTap
simpletrace output.

Signed-off-by: Stefan Hajnoczi <stefanha@redhat.com>
---
 Makefile                      | 1 +
 redhat/qemu-kvm.spec.template | 1 +
 2 files changed, 2 insertions(+)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 Makefile                      |    1 +
 redhat/qemu-kvm.spec.template |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/Makefile b/Makefile
index 414a26b..e3278c3 100644
--- a/Makefile
+++ b/Makefile
@@ -411,6 +411,7 @@ endif
 		$(INSTALL_DATA) $(SRC_PATH)/pc-bios/keymaps/$$x "$(DESTDIR)$(datadir)/keymaps"; \
 	done
 	$(INSTALL_DATA) $(SRC_PATH)/trace-events "$(DESTDIR)$(datadir)"
+	$(INSTALL_PROG) $(SRC_PATH)/scripts/simpletrace.py "$(DESTDIR)$(datadir)"
 	for d in $(TARGET_DIRS); do \
 	$(MAKE) -C $$d $@ || exit 1 ; \
         done
-- 
1.7.1

